packages/client-sdk-nodejs/src/config/transport/transport-strategy.ts

```diff
@@ -55,9 +53,9 @@ export interface TransportStrategy {
   withMaxClientAgeMillis(maxClientAgeMillis: number): TransportStrategy;
 
   /**
-   * @returns {number} the maximum number of concurrent requests that can be made by the client.
+   * returns the maximum number of concurrent requests that can be made by the client.
    */
-  getConcurrentRequestsLimit(): number;
+  getConcurrentRequestsLimit(): number | undefined;
```

Comment:
would you mind if we renamed this to `MaxConcurrentRequests`? just feels like it's a little more consistent with existing config options?